roles/openshift_logging/tasks/generate_secrets.yaml

```diff
@@ -1,13 +1,30 @@
 ---
+- name: Retrieving the cert to use when generating secrets for the logging components
+  slurp: src="{{generated_certs_dir}}/{{item.file}}"
```

Comment:
yes.  i THINK its because we mix context of what is running on the control and what is running on the target, but its masked since they are the same.  slurp pulls from the remote as whatever user; so the context is correct.  I think technically this is the start of how you might run this from a control machine that is not the master, assuming the cert writes were actually to happen remotely.